en/firefox/browsers/compare/opera.ftl

```diff
@@ -0,0 +1,50 @@
+compare-opera-like-firefox-opera-delivers = Like { -brand-name-firefox }, { -brand-name-opera } delivers a scrolling tab experience, which means that when you open more tabs than will fit on screen, it scrolls them off screen instead of just continuously shrinking them down. Also both { -brand-name-firefox } and { -brand-name-opera } have a screenshot tool that lets you capture a snapshot of your screen or part of the page. However, the { -brand-name-opera } tool doesn’t give you the ability to create one huge capture of the whole webpage, only the visible portion.
+
+# Variables:
+#   $attrs (string) - link to /firefox/pocket/ with additional attributes for analytics
+# "Facebook Messenger" is a brand name.
+compare-opera-opera-provides-a-lot-of-hidden = { -brand-name-opera } provides a lot of hidden utility within its simple and manageable interface. For example there’s built-in support for messaging apps, like { -brand-name-facebook } Messenger. There’s also a news reader that aggregates articles from your choice of sites and news outlets. The parallel feature to this on { -brand-name-firefox } is called <a { $attrs }>{ -brand-name-pocket }</a>. { -brand-name-pocket } is a free service for { -brand-name-firefox } account holders that makes it easy to find and save interesting articles and videos from all around the web. In addition, it recommends a variety of articles that expand your knowledge base curated by real, thoughtful humans.
+compare-opera-in-terms-of-head-to-head-utility = In terms of head to head utility, { -brand-name-opera } and { -brand-name-firefox } are close competitors. { -brand-name-opera } may have an advantage in one aspect with its compatibility with and access to { -brand-name-chrome }’s huge extension library. But one significant factor to consider is the fact that { -brand-name-opera }, because it’s built on { -brand-name-chromium }, is a processor-hungry browser with its RAM consumption comparable to { -brand-name-chrome }, which is known for its high CPU usage.
+
+# "Ex" is a former partner or spouse, as in "ex-boyfriend" or "ex-wife." Alternative: "former partner"
+compare-opera-both-firefox-and-opera-are = Both { -brand-name-firefox } and { -brand-name-opera } are compatible across every platform including { -brand-name-windows }, Mac, { -brand-name-linux }, { -brand-name-android } and { -brand-name-ios }. { -brand-name-firefox } account holders can easily sync their bookmarks, passwords, open tabs, and browsing history across all their signed into devices. The same is true for { -brand-name-opera } users with an account. However, many sites, especially old sites that haven’t been updated in years, block the latest version of { -brand-name-opera } entirely. So if visiting places like your Ex’s old blog is important, take heed, you may not be able to access some of the dustier corners of the internet if you use { -brand-name-opera }.
```

Comment:
`Mac`: shall we change to `macOS` to standardize the reference? Or we can add `Mac` to the brands.ftl.